test/segment-loader.test.js

```diff
@@ -831,7 +830,7 @@ QUnit.module('SegmentLoader', function(hooks) {
     });
 
     QUnit.test('only appends one segment at a time', function(assert) {
-      return setupMediaSource(loader.mediaSource_, loader.sourceUpdater_).then(() => {
+      return this.setupMediaSource(loader.mediaSource_, loader.sourceUpdater_).then(() => {
```

Comment:
where does `this.setupMediaSource` is coming from here? I'm not seeing where it is defined, and `setupMediaSource` is no longer being imported.

Reply:
its defined in the beforeEach/afterEach to handle the creation of the video element and then the removal.

Reply:
https://github.com/videojs/http-streaming/pull/1174/files#diff-76e6f47cbe1dfa39fb03a86becd7cc8d79ab123c00f4c1a74aec433201ae0115R70

Reply:
aaah, cool. Missed that this file used the default hooks.